server/tox.ini

```
@@ -1,41 +1,42 @@
 [tox]
-envlist = py
-skipsdist = true
-env_list = format, lint, unit
+env_list = 
+    lock
+    format
+    lint
+    unit
+no_package = true
+requires = 
+    tox-uv >=1.25.0
```

Nitpick, let's remove the whitespace for consistency here too

```suggestion
    tox-uv>=1.25.0
```